### ISSUE TITLE
A11Y. "show more" toggle doesn't work with keyboard

### ISSUE THREAD

**Issue by giancorderoortiz** [SAP/spartacus#11241]
**Setup**
3.1.0.RC2 Server Links
CCV2 (CX 2011.1):
-- b2c - https://spartacusstore.cg79x9wuu9-eccommerc1-p5-public.model-t.myhybris.cloud/electronics-spa/en/USD/
-- b2b - https://b2bspastore.cg79x9wuu9-eccommerc1-p5-public.model-t.myhybris.cloud/powertools-spa/en/USD/

 Linux (CX 2005.6):
-- b2c - http://spartacus-dev2.eastus.cloudapp.azure.com:4200/electronics-spa/en/USD/
-- b2b - http://spartacus-dev2.eastus.cloudapp.azure.com:5200/powertools-spa/en/USD/

**Steps to Reproduce (Using Keyboard Only):** 
1. Navigate to B2C Storefront
2. Search for HOME
3. From product listing page, select Home Theatre product.
4. Tab until arriving to "show more" toggle. 
5. Press Enter key. 

**Expected Results:** 
5. Toggle works and a small image of Home Theatre System is shown. Caption is changed to "show less". 

**Actual Results:** 
5. Nothing happens.

**Comment by giancorderoortiz** [comment]
![Screen Shot 2021-02-24 at 12 12 02 PM](https://user-images.githubusercontent.com/46171897/109038346-a1b50100-7699-11eb-9047-e37a3e26f79a.png)

**Comment by giancorderoortiz** [comment]
@developpeurweb please confirm steps to reproduce.

**Comment by developpeurweb** [comment]
> @developpeurweb please confirm steps to reproduce.

Confirming steps as described.
Currently not focusable.
While addressing this issue pls note this should be an action-link, meaning a **button** styled as a link since the action happens on the same page.

**Comment by steinsebastian** [comment]
Will be handled with v1.
